packages/ui/src/apps/collectionsUi/viewModules/start/Component.js

```diff
@@ -1,126 +1,128 @@
-                  heterogeneous collection of unsustainable in-house databases
-                  with a modern, web-based national collection management
-                  system.`}
-                  </p>
-                </Segment>
-              </Grid.Column>
-              <Grid.Column floated="right" width={6}>
-                <Segment padded="very">
-                  <Markdown
-                    fallbackLanguage="en"
-                    textKey="modules.start.changelog"
-                  />
-                </Segment>
-              </Grid.Column>
-            </Grid.Row>
+        <div className="ui center aligned grid">
+          <Grid
+            columns={2}
+            style={{ marginTop: '6.25em', maxWidth: '60em' }}
+            verticalAlign="middle"
+          >
+            <Grid.Column width={10}>
+              <Header
+                as="h1"
+                content={`${moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'welcome',
+                })} ${capitalizeFirstLetterOfEachWord(
+                  username
+                )}! ${moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'startPageText',
+                })}`}
+                style={{
+                  color: '#000000a0',
+                  fontWeight: 200,
+                  lineHeight: '2.5rem',
+                }}
+              />
+              <div
+                style={{
+                  marginBottom: '1em',
+                  marginTop: '2em',
+                }}
+              >
+                <NavLink to="/app/specimens/mammals/create/sections/0">
+                  <Button primary size="huge" type="button">
+                    <Icon name="plus" />
+                    {moduleTranslate({
+                      capitalize: true,
+                      module: 'start',
+                      textKey: 'btnRegisterMammal',
+                    })}
+                  </Button>
+                </NavLink>
+                <NavLink to="/app/specimens/mammals">
+                  <Button basic color="blue" size="huge" type="button">
+                    <Icon name="search" />
+                    {moduleTranslate({
+                      capitalize: true,
+                      module: 'start',
+                      textKey: 'btnFindMammals',
+                    })}
+                  </Button>
+                </NavLink>
+                <br />
+              </div>
+              <div
+                style={{
+                  color: '#000000a0',
+                }}
+              >
+                {moduleTranslate({
+                  capitalize: true,
+                  module: 'start',
+                  textKey: 'needHelp',
+                })}{' '}
+                <a href="mailto:team@mail.dina-web.net">
+                  dina-collections@nrm.se
```

Comment:
use the actual email address: team@mail.dina-web.net

what's in the sketch is just an example

Reply:
It is a generic contact address setup for dina related issues.